### DISCUSSION TITLE
Automatic Edge Handle Positioning

### DISCUSSION THREAD

**Discussion by trebor** [xyflow/xyflow#1834] [New Features]
I would love to have an option where the handle positions were calculated automatically.  These could even be recalculated and updated as the user drags nodes around.  The math feels relatively straight-forward and it significantly reduce overhead of me manually trying to figure out which are the two closest edges between two nodes.

I suspect this has been suggested / discussed elsewhere but I couldn't find it - so please forgive me if this is a dup.

**Comment by moklick** [comment]
Hey @trebor 

is this what you are looking for https://reactflow.dev/examples/floating-edges ?

**Comment by trebor** [comment]
wow, yes!  i'll roll this into my project.  thank you!

to me, this feels like a great default behavior or at least a built-in option for the library.  in any case, this is lovely.

**Reply by hgeldenhuys** [reply]
The problem with that example is, it only works for single 1-1 edges, if you add a second edge to the same set of nodes, the edges are on top of each other. 

I've built something that supports edge spacing, but I am not able to update the edges automatically when I move the node around

**Reply by brampurnot** [reply]
I'd be interested in this too. Any examples you can share?

**Reply by dkachur1** [reply]
Any solution to this ?
